documentation/src/main/asciidoc/topics/proc_registering_sci_remote_caches.adoc

```diff
@@ -0,0 +1,44 @@
+[id='registering-sci-remote-caches_{context}']
+= Registering serialization context initializers with {brandname} Server
+
+You must register your Protobuf schema with {brandname} Server before you can marshall user types to and from remote caches.
+
+.Prerequisites
+
+* Generate a `SerializationContextInitializer` implementation with the ProtoStream processor.
++
+After the build completes, you can find generated Protobuf schema in the `target/<schemaFilePath>/` directory.
+
+.Procedure
+
+Do one of the following to register your generated Protobuf schema with {brandname} Server:
+
+* Add the generated Protobuf schema to the `\___protobuf_metadata` cache with the {brandname} command line interface (CLI) or REST interface.
++
+This method ensures that {brandname} automatically distributes your schema across the cluster.
+
+* Use the generated `SerializationContextInitializer` implementation with a Hot Rod client to register the Protobuf schema, as in the following example:
++
+[source,java,options="nowrap",subs=attributes+]
+----
+include::code_examples/LibraryInitializerClientSchema.java[]
+----
+
+As an alternative, you can add an archive with the `SerializationContextInitializer` implementation and custom classes to the `{server_home}/server/lib` directory.
```

Comment:
This is an alternative to remote proto schema registration if only the schemas are required, i.e. for protostream -> json transcoding, however this is an essential step if protostream -> x-java-object transformations are required at any time on the server as the ProtoStream marshallers are required on the server for that type. Similarly the POJO jar(s) must be present.